test/e2e/apimachinery/resource_quota.go

```diff
@@ -136,6 +137,109 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		framework.ExpectNoError(err)
 	})
 
+	// TODO: convert the test to conformance test once the loadbalancer node port control feature is GAed
+	ginkgo.It("should create a ResourceQuota and capture the life of a load balancer type service with allocateLoadBalancerNodePorts.", func() {
+		if !feature.DefaultFeatureGate.Enabled(features.ServiceLBNodePortControl) {
```

Comment:
IIRC, I don't think this will work because test binaries do not register feature gates

Reply:
Thanks @andrewsykim! Actually the feature gate is not the main problem, the main issue I hit was, the LB type service which allocates node port can't be deleted (this behavior is consistent). I've put more details on [slack thread](https://kubernetes.slack.com/archives/C09QYUH5W/p1625009880371900), for now I doubt if it's a bug on service deletion. I tried create and delete the LB type service on my own environment, and it was working fine:
```
➜  ~ kubectl apply -f svc-lb.yaml
deployment.apps/nginx-deployment unchanged
service/nginx-lb created
➜  ~ kubectl get svc nginx-lb
NAME       TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
nginx-lb   LoadBalancer   10.100.64.51   <pending>     8080:31466/TCP   8s
➜  ~ kubectl get svc nginx-lb -o wide
NAME       TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE   SELECTOR
nginx-lb   LoadBalancer   10.100.64.51   <pending>     8080:31466/TCP   11s   app=nginx
➜  ~ time kubectl delete svc nginx-lb
service "nginx-lb" deleted
kubectl delete svc nginx-lb  0.09s user 0.04s system 73% cpu 0.184 total
➜  ~ kubectl get svc nginx-lb
Error from server (NotFound): services "nginx-lb" not found
```
Not sure what happended in the test infra that the service cannot be cleaned up. @aojea do you have any idea on how to debug the `pull-kubernetes-e2e-gce-ubuntu-containerd` environment? From the [debug log](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/100412/pull-kubernetes-e2e-gce-ubuntu-containerd/1409994251747463168#1:build-log.txt%3A37141) I can see that the LB type service is not removed, which is different from my local test results. Anything special with that environment?